### ISSUE TITLE
$page in read_method_info

### ISSUE THREAD

**Issue by kitsonk** [wkeese/old-api-viewer#25]
I am getting quite a few errors thrown in regards to variable being unassigned in generate.php.  Taking a look at the code, it is trying to figure out if something in inherited by comparing it to a non-existent variable (line 172):

``` php
    $method = array(
        "name"=>$nm,
        "scope"=>$n->getAttribute("scope"),
        "from"=>$n->getAttribute("from"),
        "visibility"=>($private=="true"?"private":"public"),
        "parameters"=>array(),
        "return-types"=>array(),
        "inherited"=>$n->getAttribute("from")!=$page,
        "constructor"=>$n->getAttribute("constructor")=="constructor"
    );
```

`$page` doesn't exist in this scope.

**Comment by wkeese** [comment]
Ah sorry about that, I checked in a fix (ef311caa7c0a6734cd1df054ce4be28689a0dba2).   Not sure why I don't see the
errors though.

**Comment by kitsonk** [comment]
Could be that your PHP is configured to not output the errors in the output
stream.  At least on my machine, any PHP errors are getting written as part
of the return text and so I see them in the application.
